ml/Config.cc

```diff
@@ -47,6 +47,11 @@ void Config::readMLConfig(void) {
     double ADWindowRateThreshold = config_get_float(ConfigSectionML, "window minimum anomaly rate", 0.25);
     double ADDimensionRateThreshold = config_get_float(ConfigSectionML, "anomaly event min dimension rate threshold", 0.05);
 
+    double SaveAnomalyPercentageEvery = config_get_number(ConfigSectionML, "window size of anomaly bit counting for anomaly percentage", 15 * 60);
```

Comment:
I don't think we want to expose this option. What if someone starts the agent multiple times with different values?

Reply:
OK.